scoop/src/main/java/com/lyft/scoop/Router.java

```diff
@@ -82,7 +92,7 @@ public void resetTo(Screen screen) {
         resetTo(screen, TransitionDirection.EXIT);
     }
 
-    public void replaceAllWith(List<Screen> screens) {
+    public void replaceAllWith(Screen... screens) {
```

Comment:
@itspbj Im not a fun of "...". We can try to add as extra method that will reuse replaceAllWith(List..)